### PR TITLE
Update GCS cloud storage credential document

### PR DESCRIPTION
**Pull request by karan6181**
## Description of changes:
- Update GCS cloud storage credential doc on how to setup GCS key and secret since the official documentation is hard to follow.

## Merge Checklist:
_Put an `x` without space in the boxes that apply. If you are unsure about any checklist, please don't hesitate to ask. We are here to help! This is simply a reminder of what we are going to look for before merging your pull request._

### General
- [x] I have read the [contributor guidelines](https://github.com/mosaicml/streaming/blob/main/CONTRIBUTING.md)
- [x] This is a documentation change or typo fix. If so, skip the rest of this checklist.
- [ ] I certify that the changes I am introducing will be backward compatible, and I have discussed concerns about this, if any, with the MosaicML team.
- [ ] I have updated any necessary documentation, including [README](https://github.com/mosaicml/streaming/blob/main/README.md) and [API docs](https://github.com/mosaicml/streaming/tree/main/docs) (if appropriate).

### Tests
- [x] I ran `pre-commit` on my change. (check out the `pre-commit` section of [prerequisites](https://github.com/mosaicml/streaming/blob/main/CONTRIBUTING.md#prerequisites))
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate).
- [x] I ran the tests locally to make sure it pass. (check out [testing](https://github.com/mosaicml/streaming/blob/main/CONTRIBUTING.md#running-tests))
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes.

### CODE REVIEW

File: docs/source/how_to_guides/configure_cloud_storage_cred.md [hunk 1]
```diff
@@ -56,7 +56,9 @@ For [MosaicML platform](https://www.mosaicml.com/cloud) users, follow the steps
 
 ### Others
 
-Streaming dataset supports [GCP user credentials](https://cloud.google.com/storage/docs/authentication#user_accounts).  Users must set their GCP `user access key` and GCP `user access secret` in the run environment.
+Streaming dataset supports [GCP user credentials](https://cloud.google.com/storage/docs/authentication#user_accounts) or [HMAC keys for User account](https://cloud.google.com/storage/docs/authentication/hmackeys).  Users must set their GCP `user access key` and GCP `user access secret` in the run environment.
+
+From Google Cloud Console Navigate to `Google Storage` > `Settings (Left vertical pane)` > `Interoperability` > `Service account HMAC` > `User account HMAC` > `Access keys for your user account` > `Create a key`.
```
Comment (knighton) [inline docs/source/how_to_guides/configure_cloud_storage_cred.md:61]: From the Google Cloud console, navigate

**Review by knighton — APPROVED** [review]

**Review by ssovine — APPROVED** [review]
Looks good to me once James change is integrated. Nice job!

File: docs/source/how_to_guides/configure_cloud_storage_cred.md (hunk 1 above)
Comment (karan6181) [reply docs/source/how_to_guides/configure_cloud_storage_cred.md]: Addressed in https://github.com/mosaicml/streaming/pull/181/commits/4f5a1d61ae90a392a183a3380cec6c3814dc9adf.
